src/promptflow-devkit/promptflow/_sdk/_load_functions.py

```diff
@@ -81,7 +82,13 @@ def load_flow(
     :return: A Flow object
     :rtype: ~promptflow._sdk.entities._flows.Flow
     """
-    return Flow.load(source, **kwargs)
+    flow = Flow.load(source, **kwargs)
+    if check_legacy_llm_in_flow_nodes(flow._data):
+        logger.warning(
+            "Please upgrade to the latest version of promptflow-tools to consume new LLM tools. "
```

Comment:
Besides upgrade local pf-tools version, I think there are other things people also need to do to dismiss the warning.